package.json

```diff
@@ -11,7 +11,8 @@
     "serve": "npx fastify start ./src/routes.js",
     "dev": "sh -c 'FASTIFY_LOG_LEVEL=debug FASTIFY_WATCH=true FASTIFY_SWAGGER=true npm run serve'",
     "test": "npx mocha ./src/test.js",
-    "coverage": "npx c8 --100 --exclude src/test.js npm run test"
+    "coverage": "npx c8 --100 --exclude src/test.js npm run test",
+    "start": "node /usr/src/app/src/server.js"
```

Comment:
I removed `server.js` in my branches (and changed `npm serve` to `npm start`). If we really need it, we should have a `cloudRun.js`, and the command here should be `npm cloudRun`.

Reply:
you will be glad to know that we don't need npm cloudRun

Reply:
So we use `npm start` just like in `master`, with the `fastify` CLI?

Reply:
for local run we use npm start,

when deploying to gcp, we just run it by
CMD ["node", "/usr/src/app/src/cloudRun.js"]
inside the docker

Reply:
Ah, wonderful.